easybuild/easyconfigs/c/CubeWriter/CubeWriter-4.4.3-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,51 @@
+# #
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2019 Juelich Supercomputing Centre, Germany
+# Authors::   Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+# #
+
+easyblock = 'EB_Score_minus_P'
+
+name = 'CubeWriter'
+version = '4.4.3'
+
+homepage = 'https://www.scalasca.org/software/cube-4.x/download.html'
+description = """
+ Cube, which is used as performance report explorer for Scalasca and Score-P,
+ is a generic tool for displaying a multi-dimensional performance space
+ consisting of the dimensions (i) performance metric, (ii) call path, and
+ (iii) system resource. Each dimension can be represented as a tree, where
+ non-leaf nodes of the tree can be collapsed or expanded to achieve the
+ desired level of granularity.
+
+ This module provides the Cube high-performance C writer library component.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://apps.fz-juelich.de/scalasca/releases/cube/%(version_major_minor)s/dist']
+sources = ['cubew-%(version)s.tar.gz']
+checksums = ['93fff6cc1e8b0780f0171ef5302a2e1a257f99b6383fbfc1b9b82f925ceff501']
+
+builddependencies = [
+    ('GCC', '%(toolchain_version)s', '', True),
```

Comment:
Please use `binutils` as the build dependency here and in the other easyconfigs in this PR and #13288.

Reply:
Doing that for now but I'd appreciate a long term solution here :)

Reply:
Thanks and I agree that it would be good to get a better long term solution for this.